packages/app/pages/guide.tsx

```diff
@@ -0,0 +1,107 @@
+import { Collapse, Layout } from 'antd'
+import NavBar from '../components/Nav/NavBar'
+import 'antd/dist/antd.css'
+
+const { Panel } = Collapse
+const { Content, Footer } = Layout
+
+const GuidePage = () => {
+  return (
+    <Layout className="layout">
+      <NavBar />
+
+      <Content style={{ padding: '0 50px' }}>
+        <div className="site-layout-content">
+          <section id="welcome">
+            <h1>Welcome to HelpMe</h1>
+            <h2>For Instructors</h2>
+            <h3> Video to get started</h3>
+            <iframe
+              width="560"
+              height="315"
+              src="https://www.youtube.com/embed/RjNERqpIQaw"
```

Comment:
This video is outdated for new UI.
Could be confusing for new users.

Reply:
This is true though it seems the flow is mostly the same. Also, seeing as how we'll be making a lot more UI changes as the semester goes on, it might make sense to maybe leave updating the video for another time (and who knows, maybe if we need to make a video for our thesis or something, we could use that as the video)